core/src/main/scala/com/salesforce/op/stages/impl/feature/PercentileCalibrator.scala

```diff
@@ -61,6 +61,7 @@ class PercentileCalibrator(uid: String = UID[PercentileCalibrator])
       .setNumBuckets($(expectedNumBuckets))
       .setRelativeError(0)
       .setInputCol(dataset.columns(0))
+      .setOutputCol(dataset.columns(0) + "-out")
```

Comment:
shouldnt the output column be set to the output name from the wrapping class?

Reply:
Its internal to the estimator, so it does not really matter that the names are.

Reply:
but output column is now a required field?

Reply:
yes ;)